app/assets/javascripts/shared/datatable/state.js

```diff
@@ -72,7 +74,7 @@ DradisDatatable.prototype.updateRecordState = function (newState) {
       data: {
         ids: DradisDatatable.prototype.rowIds(selectedRows),
         state: newState,
-        return_to: $('[data-behavior~=qa-viewer]').length > 0 ? 'qa' : null
+        return_to: $('[data-behavior~=qa-viewer]').length > 0 ? 'qa' : null,
```

Comment:
does this need to go?

Reply:
Replaced with a data attribute instead ✅